Bio/Blast/NCBIXML.py

```diff
@@ -222,14 +224,14 @@ def _end_Iteration(self):
 
     # Header
     def _end_BlastOutput_program(self):
-        """BLAST program, e.g., blastp, blastn, etc.
+        """BLAST program, e.g., blastp, blastn, etc.(PRIVATE).
```

Comment:
You need a space before the (PRIVATE) text, but since ``etc`` is an abbreviation that means I would suggest this:

```
   """BLAST program, e.g., blastp, blastn, etc. (PRIVATE).
```